jenkins/internal-ci/generic/pr/cortx_deployment/csm-agent.groovy

```diff
@@ -0,0 +1,267 @@
+                        sed -i -e 's/BRANCH:.*/BRANCH: "csm-agent-pr"/g' ${CORTX_ISO_LOCATION}/RELEASE.INFO
+                    popd
+
+                    cp "${THIRD_PARTY_LOCATION}/THIRD_PARTY_RELEASE.INFO" "${DESTINATION_RELEASE_LOCATION}"
+                    cp "${CORTX_ISO_LOCATION}/RELEASE.INFO" "${DESTINATION_RELEASE_LOCATION}"
+
+                    cp "${CORTX_ISO_LOCATION}/RELEASE.INFO" .
+                '''    
+
+                archiveArtifacts artifacts: "RELEASE.INFO", onlyIfSuccessful: false, allowEmptyArchive: true    
+            }
+
+        }
+
+        stage ("Build CORTX-ALL image") {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {
+                    try {
+                        def buildCortxAllImage = build job: 'cortx-docker-images-for-PR', wait: true,
+                            parameters: [
+                                string(name: 'CORTX_RE_URL', value: "${CORTX_RE_URL}"),
+                                string(name: 'CORTX_RE_BRANCH', value: "${CORTX_RE_BRANCH}"),
+                                string(name: 'BUILD', value: "${CORTX_BUILD}"),
+                                string(name: 'OS', value: "${OS_VERSION}"),
+                                string(name: 'CORTX_IMAGE', value: "${CORTX_IMAGE}"),
+                                string(name: 'GITHUB_PUSH', value: "yes"),
+                                string(name: 'TAG_LATEST', value: "no"),
+                                string(name: 'DOCKER_REGISTRY', value: "cortx-docker.colo.seagate.com"),
+                                string(name: 'EMAIL_RECIPIENTS', value: "DEBUG")
+                            ]
+                        env.cortx_all_image = buildCortxAllImage.buildVariables.cortx_all_image
+                        env.cortx_rgw_image = buildCortxAllImage.buildVariables.cortx_rgw_image
+                    } catch (err) {
+                        build_stage = env.STAGE_NAME
+                        error "Failed to Build CORTX-ALL image"
+                    }
+                }
+            }
+        }
+
+        stage ("Deploy Cortx Cluster") {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {
+                    build job: "K8s-1N-deployment", wait: true,
+                    parameters: [
+                        string(name: 'CORTX_RE_URL', value: "${CORTX_RE_URL}"),
+                        string(name: 'CORTX_RE_BRANCH', value: "${CORTX_RE_BRANCH}"),
+                        string(name: 'CORTX_ALL_IMAGE', value: "${env.cortx_all_image}"),
+                        string(name: 'CORTX_SERVER_IMAGE', value: "${env.cortx_rgw_image}"),
+                        string(name: 'CORTX_SCRIPTS_REPO', value: "Seagate/cortx-k8s"),
+                        string(name: 'CORTX_SCRIPTS_BRANCH', value: "integration"),
```

Comment:
Please use services release v0.1.0

Reply:
Check if you can use default so we do not need to change it at multiple locations